get_socrata_data.py

```diff
@@ -0,0 +1,23 @@
+from sodapy import Socrata
+import pandas as pd
+
+cities: "dict[str, str]" = {"data.cityofnewyork.us":"dm9a-ab7w",
+        "data.lacity.org":"nbyu-2ha9",
+        "data.austintexas.gov": "3syk-w9eu",
+        "data.cityofchicago.org": "ydr8-5enu",
+        "data.mesaaz.gov": "2gkz-7z4f"}
+
+def get_data():
```

Comment:
This function should take in city and the associated string as an argument, then pass in values from the cities dict into the function.

Reply:
👍